spec/lib/hubspot/blog_spec.rb

```diff
@@ -1,116 +1,150 @@
 require 'timecop'
 
 describe Hubspot do
-  let(:example_blog_hash) do
-    VCR.use_cassette("blog_list", record: :none) do
-      url = Hubspot::Connection.send(:generate_url, Hubspot::Blog::BLOG_LIST_PATH)
-      resp = HTTParty.get(url, format: :json)
-      resp.parsed_response["objects"].first
-    end
-  end
-  let(:created_range_params) { { created__gt: false, created__range: (Time.now..Time.now + 2.years)  } }
-
   before do
     Hubspot.configure(hapikey: "demo")
-    Timecop.freeze(Time.local(2012, 'Oct', 10))
+    Timecop.freeze(Time.utc(2012, 'Oct', 10))
   end
 
   after do
     Timecop.return
   end
 
   describe Hubspot::Blog do
-
     describe ".list" do
-      cassette "blog_list"
-      let(:blog_list) { Hubspot::Blog.list }
+      it "returns a list of blogs" do
+        VCR.use_cassette("blog_list", record: :none) do
```

Comment:
For anyone new to `VCR` (like me), the `record: :none` will raise anytime VCR can't find a match for the request. This prevents VCR from issuing a real request when the request doesn't match an recorded request.